medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.json

```diff
@@ -9,8 +9,10 @@
       }
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "cms": "https://rsbc-dfp-cms-adapter-dev.apps.silver.devops.gov.bc.ca/"
+
```

Comment:
We've been keeping this type of configuration out of git, can you remove this please?

Reply:
@GeorgeWalker if you want something out of git, is it not better we add those in the .gitignore file?